### DISCUSSION TITLE
Enhancement: Chat privacy (encryption)

### DISCUSSION THREAD

**Discussion by tc-platform** [danny-avila/LibreChat#3628] [Feature Requests & Suggestions]
### What features would you like to see added?

Hello,
as admin i can easily connect to e.g. the mongodb and can have a look what users are asking. 

It would be awesome if conversations are NOT stored in cleartext.

### More details

Instead it would be nice if they are encrypted with a password a user sets itself to enable better data privacy for the enduser or are just saved locally in browser storage.

### Which components are impacted by your request?

General

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
This is not planned. It would add too much overhead, and even MongoDB itself does volume encryption at rest for remote services (atlas). There are ways to protect the data with similar encryption methods locally, but none in such a way that "protects" the data from the admin.
